doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
  */
 ////
 
-calcurse(1)
+CALCURSE(1)
 ===========
 
-Name
+NAME
 ----
 
-calcurse - text-based organizer
+calcurse - terminal-based organizer for interactive and command line use
 
-Synopsis
+SYNOPSIS
 --------
 
-[verse]
-'calcurse' *-Q* [options] [*--from* <date>] [*--to* <date>|*--days* <num>]
-'calcurse' *-G* [options]
-'calcurse' *-i*<file>
-'calcurse' *-x*<format>
-'calcurse' *--gc*
-'calcurse' *--status*
-'calcurse' *--version*
-'calcurse' *--help*
-
-Description
------------
+--
+*calcurse* [-D 'directory'] | [-C 'confdir'] | [-c 'calendar-file']
+
+*calcurse* -Q [*--from* 'date'] [*--to* 'date' | *--days* 'number']
+
+*calcurse* -a | -d 'date' | -d 'number' | -n | -r['number'] | -s['date'] | -t['number']
+
+*calcurse* -h | --status | -G | -g | -i 'file' | -P | -x['file'] | --daemon
+--
 
-Calcurse is a text-based calendar and scheduling application. It helps keeping
-track of events, appointments and everyday tasks.  A configurable notification
-system reminds user of upcoming deadlines, and the curses based interface can
-be customized to suit user needs.  All of the commands are documented within an
-online help system.  
+Briefly, the first form shows how to invoke calcurse interactively; the
+remaining are command line forms. The second form shows queries, the third
+short-form queries; both may be combined with filter options as well as
+format options. The fourth form is miscellaneous help and support functions.
+All details are in <<_options,OPTIONS>>.
 
-Options
+DESCRIPTION
+-----------
+
+Calcurse is a calendar and scheduling application for use in a terminal
+session (terminal emulator). When invoked without options, calcurse enters
+interactive mode; in most other cases, calcurse produces output on the
+terminal and exits.  It helps keeping track of events, appointments and
+everyday tasks. Interactive mode is used when data are entered or when already
+existing entries are inspected or edited. All data are saved to disk as text
+files. Command line mode is used for queries and administrative tasks and for
+automating tasks in scripts.
+
+The interactive interface is based on ncurses and can be customized to suit
+user preferences. Customization includes program behaviour as well as visual
+appearance and key bindings, and is performed interactively; the result is
+automatically saved to disk and loaded on every invocation. Available actions
+are documented in an online help system. A configurable notification system
+issues reminders of upcoming deadlines.
+
+When leaving the interactive program, a background daemon may continue running
+and issue notifications; it stops automatically when the interactive mode is
+reentered.
+
+This man-page mainly describes the command-line mode.
+
+OPTIONS
 -------
 
-The following options are supported:
+.Input and Output Date Format
+*****************************
+Many options require a 'date' argument, and query results per day are set apart 
+by a leading 'date line'.
```

Comment:
I see what you intended with this introductory section but I am not sure whether it is all that useful. A man page is used like a dictionary most of the time and rarely read from cover to cover. Maybe we should move all this to the formatting section? As a user, I'd expect to find it there.